app/services/site_channel_domain_setter.rb

```diff
@@ -13,22 +13,23 @@ def perform
   private
 
   def normalize_domain
-    require "faraday"
+    require 'addressable'
 
-    channel_details.brave_publisher_id = SiteChannelDomainNormalizer.new(domain: channel_details.brave_publisher_id_unnormalized).perform
+    unless channel_details.brave_publisher_id_unnormalized.starts_with?("http")
```

Comment:
Explicitly check for "http://" or "https://" otherwise domains that start with "http", i.e "http-tips.com" will not get the required protocol added.